app/src/components/market/market_wizard_creator.tsx

```diff
@@ -56,7 +56,7 @@ export const MarketWizardCreator = (props: Props) => {
         probability: 50,
       },
     ],
-    loadedQuestion: false,
+    loadedQuestion: '',
```

Comment:
I don't think it's a good idea to use an empty string to signal a missing value. If the goal is to use this variable to indicate that the question is loaded *and* to have the question id, it should probably be renamed and retyped to `loadedQuestionId: Maybe<string>`.

Reply:
Done